src/fiatExchanges/navigator.tsx

```diff
@@ -1,16 +1,5 @@
-import { navigate, navigateHome } from 'src/navigator/NavigationService'
-import { Screens } from 'src/navigator/Screens'
-import { getExperimentParams } from 'src/statsig'
-import { ExperimentConfigs } from 'src/statsig/constants'
-import { StatsigExperiments } from 'src/statsig/types'
+import { navigateHome } from 'src/navigator/NavigationService'
 
-export const navigateToFiatExchangeStart = () => {
-  if (
-    getExperimentParams(ExperimentConfigs[StatsigExperiments.HOME_SCREEN_ACTIONS])
-      .showAddWithdrawOnMenu
-  ) {
-    navigate(Screens.FiatExchange)
-  } else {
-    navigateHome()
-  }
+export function navigateToFiatExchangeStart() {
+  navigateHome()
```

Comment:
We could remove that intermediary function now. Let me know if you prefer that.

Reply:
that sounds clearer to me (removing the intermediary)